dockerfiles/spoke.Dockerfile

```diff
@@ -1,6 +1,7 @@
 # syntax=docker/dockerfile:1
 ARG NODE_VERSION=16.13
 
-FROM --platform=linux/amd64 node:${NODE_VERSION} AS dev
+FROM --platform=linux/amd64 node:${NODE_VERSION}
 HEALTHCHECK CMD curl -fk https://localhost:9090 || exit 1
 WORKDIR /code
+ENTRYPOINT ["./scripts/run-local-reticulum.sh"]
```

Comment:
I'm not familiar enough with docker to know if the following concern matters, but...

It seems that using `ENTRYPOINT` instead of `CMD` means that I won't be able to override what runs on start by modifying the `command:` entry in `docker-compose.yml`. I find myself doing this frequently (usually to change it to `sleep infinity` so that I can `exec -it <service_name> bash` and run things interactively). It would be nice if we could keep this property of the dockerfiles / setup.

Reply:
I’ll switch it back to `CMD` then.

P.S. If you ever want to spin up a shell on spoke without starting the whole world, you can just run **$** `mutagen-compose run --rm spoke bash`.